plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/dlf/AlibabaDlfMetaStoreConfig.java

```diff
@@ -0,0 +1,631 @@
+public class AlibabaDlfMetaStoreConfig
+{
+    public Optional<String> hiveMetastoreAuthenticationType = Optional.empty();
+    public Optional<String> hiveMetastoreServiceClientPrincipal = Optional.empty();
+    public Optional<String> hiveMetastoreServicePrincipal = Optional.empty();
+    public Optional<String> hiveMetastoreClientKeytab = Optional.empty();
+    public Optional<Boolean> enableBitVector = Optional.empty();
+    public Optional<Integer> batchSizeForGetPartititon = Optional.empty();
+    private Optional<String> regionId = Optional.empty();
+    private Optional<String> endpoint = Optional.empty();
+    private Optional<String> logStore = Optional.empty();
+    private Optional<String> userId = Optional.empty();
+    private Optional<String> accessKeyId = Optional.empty();
+    private Optional<String> accessKeySecret = Optional.empty();
+    private Optional<String> defaultWarehouseDir = Optional.empty();
+    private Optional<AKMode> akMode = Optional.empty();
+    private boolean isNewStsMode = true;
+    private Optional<ProxyMode> proxyMode = Optional.empty();
+    private short pageSize = DataLakeConfig.DEFAULT_CATALOG_PAGE_SIZE;
+    private int batchSize = DataLakeConfig.DEFAULT_CATALOG_ACCURATE_BATCH_SIZE;
+    private int readTimeout = DataLakeConfig.DEFAULT_CATALOG_SERVER_READ_TIMEOUT_MILLS;
+    private int connTimeout = DataLakeConfig.DEFAULT_CATALOG_SERVER_CONN_TIMEOUT_MILLS;
+    private boolean deleteDirWhenDropSchema = DataLakeConfig.DELETE_DIR_WHEN_DROP_SCHEMA_VALUE;
+    private Optional<Boolean> dbCacheEnable = Optional.empty();
+    private Optional<Integer> dbCacheSize = Optional.empty();
+    private Optional<Integer> dbCacheTTLMins = Optional.empty();
+    private Optional<Boolean> tbCacheEnable = Optional.empty();
+    private Optional<Integer> tbCacheSize = Optional.empty();
+    private Optional<Integer> tbCacheTTLMins = Optional.empty();
+    private Optional<Boolean> assumeCanonicalPartitionKeys = Optional.empty();
+
+    private Optional<Boolean> enableFileOperation = Optional.empty();
+    private Optional<Double> enableFileOperationGrayRate = Optional.empty();
+    private Optional<Boolean> enableRecordActionLog = Optional.empty();
+    private Optional<Boolean> enableRecordLog = Optional.empty();
+    private Optional<Integer> tableColStatsBatchSize = Optional.empty();
+
+    public Optional<String> getRegionId()
+    {
+        return regionId;
+    }
+
+    @Config(DataLakeConfig.CATALOG_REGION_ID)
+    @ConfigDescription("Region Id for Data Lake Formation Catalog")
+    public AlibabaDlfMetaStoreConfig setRegionId(String regionId)
+    {
+        if (regionId == null || regionId.isEmpty()) {
+            this.regionId = Optional.of("cn-hangzhou");
+        }
+        else {
+            this.regionId = Optional.of(regionId);
+        }
+        return this;
+    }
+
+    public Optional<String> getEndpoint()
+    {
+        return endpoint;
+    }
+
+    @Config(DataLakeConfig.CATALOG_ENDPOINT)
```

Comment:
The config names should IMO be stated explicitly, not inherited from `com.aliyun.datalake.metastore.common.DataLakeConfig`

In Aliyun's context, the prefix `dlf.` is meaningful, but in Trino context it's not.

Reply:
Done. Some api are fixed, so I have to do this kind of revise here. https://github.com/trinodb/trino/pull/11239/files#r826875881